code/Data exploration.R

```diff
@@ -66,6 +98,13 @@ data <- data %>%mutate(SEASON = case_when(
   MONTH %in%  c(12, 1, 2)  ~ "SUMMER",
   MONTH %in%  3:5  ~ "AUTUMN",
   TRUE ~ "WINTER"));
+
+##Season indicator
+data$SPRING <- if_else(data$SEASON == 'SPRING',1,0)
+data$SUMMER <- if_else(data$SEASON == 'SUMMER',1,0)
+data$AUTUMN <- if_else(data$SEASON == 'AUTUMN',1,0)
+data$WINTER <- if_else(data$SEASON == 'WINTER',1,0)
```

Comment:
we have to be careful with dummy variables, usually we leave one out to avoid correlation

Reply:
Definitely, will keep in mind